.github/workflows/check_author_lockfile.yaml

```diff
@@ -0,0 +1,30 @@
+name: "Check Author Lockfile"
+
+on:
+  workflow_call:
+    inputs:
+      runs_on:
+        required: false
+        type: string
+        default: self-hosted
+
+      lockfile_author:
+        required: true
+        type: string
+        default: 'none'
+      
+      action:
+        required: true
+        type: string
+
+jobs:
+  check_author_lockfile:
+    name: Check Author Lockfile
+    runs-on: ${{ inputs.runs_on }}
+    steps:
+      - name: echo
+        run: |
+          echo "lockfile_author=${{ inputs.lockfile_author }}"
```

Comment:
https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/

Improve it using job summaries